connectors/overview.mdx

```diff
@@ -95,6 +95,25 @@ CREATE TABLE my_source (
 )
 ```
 
+## Bad Data
+
+The `bad_data` option allows you to configure how Arroyo handles records that fail
```

Comment:
Might be worth mentioning where it is configured when creating a source in the UI (part of defining the schema).